### PR TITLE
Fix error state of lists of not taking the full height

### LINKED ISSUE #6226: Empty states on post likes/reposts/quotes screen don't span the full height
### Steps to Reproduce

1. Find an empty post (or find a post who has blocked everyone who liked/reposted/quoted it, I suppose)
2. Manually redirect to their likes/reposts/quotes page

### Attachments

image

### What platform(s) does this occur on?

Web (Desktop)

### Device Info

Firefox 134 (nightly); Linux (Arch Linux; Linux aster 6.11.7-2-cachyos #1 SMP PREEMPT_DYNAMIC Fri, 08 Nov 2024 17:23:46 +0000 x86_64 GNU/Linux)

### What version of the app are you using?

Build version: 1.94.0; Bundle info: dev (prod); Bundle date: 24111110; Platform: web; Platform version: undefined

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by khuddite**
Fixes #6226 

1. Summary
    It seems there was an inconsistency between the loading state and error/empty state styles in the code. Specifically, the loading state has [a.h_full_vh,](https://github.com/khuddite/social-app/blob/05aef7940cd0dd1e45f8fa54ebfd758206599677/src/components/Lists.tsx#L181) whereas error/empty state [doesn't](https://github.com/bluesky-social/social-app/blob/57464a8fe0ccdbc9cde0dedf7f274d969491cb88/src/components/Error.tsx#L34). Instead, it has `a.flex_1` which sets the view height to its content height.
   
2. Solution
    Updated `Error` component to have `a.h_full_vh` in its style.

3. Screenshots
Screenshot 2024-11-11 at 10-13-28 Post by @alicewastaken bsky social — Bluesky
Screenshot 2024-11-11 at 10-13-36 Post by @alicewastaken bsky social — Bluesky

### CODE REVIEW

**Comment by gaearon** [comment]
is this for the same issue as https://github.com/bluesky-social/social-app/pull/6235 or a different one?

**Comment by khuddite** [comment]
> is this for the same issue as #6235 or a different one?

I'd say they are not necessarily the same, but they are definitely related. I was actually about to test my changes on mobile devices, but I couldn't quite figure out how to navigate to the `reposted by` screen from the post screen.

But for now, it seems we need to get rid of `a.flex_1` and add both `a.w_full` and `a.h_full_vh`.

I can investigate more if someone can help me navigate to `Reposted by` screen on mobile.

**Comment by khuddite** [comment]
Alright, so I was able to debug it on my real Android device via deep link. It seems we don't necessarily need `a.w_full` to fix the issue #6235 is supposed to fix. Just removing `a.flex_1` will fix the issue. Actually, I think not adding it will be better for consistency because the loading state doesn't have that style either. https://github.com/bluesky-social/social-app/blob/57464a8fe0ccdbc9cde0dedf7f274d969491cb88/src/components/Lists.tsx#L179

However, it's required to add `a.h_full_vh` style for this PR to fix the reported issue. And that aligns with the error state styles because it has `a.h_full_vh`, too.

So, in summary, I think having `a.flex_1` was a mistake in the first place, and that should be replaced with `a.h_full_vh` to fix both issues. And #6235 can be closed in favor of this PR.

**Comment by khuddite** [comment]
I seems this PR has a conflict now, and it needs to be rebased off of the main branch. However, if I force push after rebase, I believe it will clear the chat history. Please advise what's the best way to deal with it. Thanks!

**Comment by mozzius** [comment]
Hi - just rebase, I'm happy to merge this once there are no conflicts!

**Review by mozzius — APPROVED** [review]

**Comment by khuddite** [comment]
I am sorry, but I am not really familiar with `git rebase`. It seems merge conflicts are already gone.

Please advise if there is anything left for me to do.

**Comment by mozzius** [comment]
no I just fixed the conflicts manually! nothing to do. thanks for the contribution!
